README.md

```diff
@@ -131,3 +131,30 @@ We are using the dev environment to integrate all the approved code from master
 1. Test that everything still works on Test.
 
 Once production is live, we should plan to deploy to production at least once per sprint—if not more frequently.
+
+## Local build process for theme files
+
+There is a separate build process that is run in local development environments, to compile template files such as CSS and JS. The resulting files, in addition to the modified source files, are committed to the repo and deployed to upstream environments. They are always built locally, never as part of the main build process we've implemented with CircleCI.
+
+### Install dependencies 
+
+1. Install Node.js using the Mac or Windows installer available on nodejs.org.
+1. Open a terminal window (PowerShell on Windows) and run "npm install -g gulp-cli" to install gulp.js
+1. Navigate to the project theme directory (portlandor/web/themes/custom/cloudy)
+1. Install project dependencies by running "npm install"
+
+### Using gulp.js to build CSS files
+
+1. Make modifications to the desired scss files in the theme. Never modify styles.css directly.
+1. In the root directory of the theme, run "gulp sass" to build just the CSS a single time.
```

Comment:
You could also run `gulp js` to get just the Javascript changes. You may want to add that as well.